CHANGELOG/CHANGELOG-0.2.md

```diff
@@ -0,0 +1,26 @@
+# Changelog 0.2
+
+## 0.2.0 - 2022-08-17
+
+### Added
+
+- E2e tests for Theia functions. ([#66](https://github.com/antrea-io/theia/pull/66), [@yanjunz97]) ([#71](https://github.com/antrea-io/theia/pull/71), [@heanlan]) ([#77](https://github.com/antrea-io/theia/pull/77), [@dreamtalen])
+  * Provide e2e tests for Theia components (ClickHouse monitor and Grafana dashboard)
+  * Improve policy recommendation e2e test
+- ClickHouse cluster support and data schema management. ([#55](https://github.com/antrea-io/theia/pull/55), [@yanjunz97]) ([#81](https://github.com/antrea-io/theia/pull/81), [@yanjunz97])
+  * Introduce support for Clickhouse cluster deployment with shards and replicas
+  * Introduce version control to Clickhouse data schema, which detects the data  schema version in initializing stage
```

Comment:
```suggestion
  * Introduce version control to ClickHouse data schema, which detects the data schema version in initializing stage
```
Do we need to mention some necessary upgrade would also be applied automatically in initializing stage?